### ISSUE TITLE
trainA&trainB.size 

### ISSUE THREAD

**Issue by Allex-shell** [linjx-ustc1106/TuiGAN-PyTorch#9]
When i have put two different style images in datas/trainA&trainB randomly, run code ,but got a error ,''The size of tensor a (150) must match the size of tensor b (35) at non-singleton dimension 3'',shoud i resize the source images what it was needed.Novice consult.Thanks.

**Comment by linjx-ustc1106** [comment]
Yes, currently our model assumes that source image and target image have the same size.
